### ISSUE TITLE
bug: datetime input for python broken

### ISSUE THREAD

**Issue by TheOneValen** [windmill-labs/windmill#2223]
### Describe the bug

I have a python script that takes a datetime as input. The auto-ui correctly shows me the nice selection ui. When I run the script, I get the following error:

```ExecutionErr: ExitCode: 1, last log lines:
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/tmp/windmill/wk-ddd2b84fcdec-Akmar/018a5a43-9cfe-c80a-6ea3-dbb49752e8e1/wrapper.py", line 15, in <module>
    kwargs["query_from"] = datetime.strptime(kwargs["query_from"], '%Y-%m-%dT%H:%M')
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/_strptime.py", line 352, in _strptime
    raise ValueError("unconverted data remains: %s" %
ValueError: unconverted data remains: :00.000Z
```

This worked before, I don't know in which version this broke.

### To reproduce

1. Make a python script with datetime input
2. Run it

### Expected behavior

Datetime getting parsed correctly.

### Application version

1.165

**Comment by rubenfiszel** [comment]
Thanks for the report, fixed in latest

**Comment by TheOneValen** [comment]
Can confirm this is fixed in 166.1 :)
